library/src/main/res/values/attrs.xml

```diff
@@ -2,5 +2,7 @@
 <resources>
     <declare-styleable name="KujakuMapView">
         <attr name="current_location_btn_visibility" format="boolean" />
+        <attr name="add_btn_visibility" format="boolean" />
```

Comment:
Do you think we really need this option? If so, can we have it work as on unit(add & done btn)
Or better yet, can the usage if this be clearly provided or implemented since exposing such functionality also calls for provision of callbacks to consume the related actions/events -> since the actions/events need to be consumed by the third party app

Reply:
Is the suggestion here NOT to allow the library users to have the ability to set the button to `visible`?

Reply:
Yes and no

Reply:
Yes and no. Read for more explanation

> Do you think we really need this option? Why? If so, can we have it work as on unit (add & done btn)
Or better yet, can the usage of this be clearly provided or implemented since exposing such functionality also requires providing callbacks to consume the related actions/events -> since the actions/events need to be consumed by the third party app.

This also applies for `done_btn_visibility`

Reply:
I have removed the attributes.